pkg/sbproxy/sbproxy.go

```diff
@@ -139,15 +139,16 @@ func New(ctx context.Context, cancel context.CancelFunc, settings *Settings, pla
 	if err != nil {
 		return nil, fmt.Errorf("error creating notifications producer: %s", err)
 	}
-	proxyPath := settings.Reconcile.URL + APIPrefix
-	resyncer := reconcile.NewResyncer(settings.Reconcile, platformClient, smClient, proxyPath)
+	smPath := settings.Sm.URL + APIPrefix
```

Comment:
local dev (make all) will probably not work now as you need different SM URL value to be used when registering SM in CC and when the proxy docker container calls sm docker container

Reply:
Fixed.